README.md

```diff
@@ -121,3 +121,31 @@ In a standard project we use three fixed Lagoon environments: `dev`, `stage` and
 Lagoon should also be configured to create automatic environments for feature
 branches that are prefixed with `lagoon/`. Those will be filled with test
 content.
+
+## "Strangling" legacy systems
```

Comment:
I would give a very very brief explanation here and move the rest of the docs to `strangler.ts` directly. Reasons:
- There the docs will really help with development
- Less chances that docs will be outdated